lib/game.dart

```diff
@@ -137,7 +136,7 @@ class _GameRenderBox extends RenderBox with WidgetsBindingObserver {
   @override
   void paint(PaintingContext context, Offset offset) {
     context.canvas.save();
-    context.canvas.translate(game.builder.offset.dx, game.builder.offset.dy);
+    context.canvas.translate(game.builder.offset.dx + offset.dx, game.builder.offset.dy+ offset.dy);
```

Comment:
This line is key. Please, check if this wouldn't break anything @luanpotter

Reply:
Looks good to me, but please try it out with the examples in the docs folder and also flame-example or bgug!

Reply:
If you want, I can help you test on Tales @renancaraujo

Reply:
@erickzanardo would you do it?

Reply:
Yeah, could you rebase this branch with master? So I can test everything?

Reply:
@erickzanardo done